ydb/core/protos/table_service_config.proto

```diff
@@ -304,7 +304,7 @@ message TTableServiceConfig {
 
     optional uint64 QueryReplayCacheUploadTTLSec = 62 [default = 36000];
 
-    optional bool EnableQueryServiceSpilling = 63 [ default = true ];
+    optional bool EnableQueryServiceSpilling = 63 [ default = false ];
```

Comment:
Make sure there are no tag overlaps in main.

Reply:
This tag remains the same, only the default value is changed